### PR TITLE
Add Voicebank (vctk_noisy) script

### PR DESCRIPTION
**Pull request by neillu23**
README.md will be on the next PR after the training is finished.

### CODE REVIEW

File: egs2/vctk_noisy/enh1/local/vctk_data_prep.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    echo "Error: $f is not a directory."
+    exit 1;
+  fi
+done
+
+data=./data
+rm -r ${data}/tr_26spk 2>/dev/null || true
+rm -r ${data}/{cv, tt}_2spk 2>/dev/null || true
+
+tmpdir=data/temp
```
Comment (Emrys365) [inline egs2/vctk_noisy/enh1/local/vctk_data_prep.sh:33]: It is better to remove $tmpdir when not used.

File: egs2/vctk_noisy/enh1/local/vctk_data_prep.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+mkdir -p $tmpdir 
+
+train_dir=${NOISY_SPEECH}/noisy_trainset_28spk_wav
+test_dir=${NOISY_SPEECH}/noisy_testset_wav
+
+echo "Building training and testing data"
+
+find $train_dir -name '*.wav' -not -name 'p226_*.wav' -not -name 'p287_*.wav' | sort -u > $tmpdir/tr.flist
+find $train_dir -name 'p226_*.wav' -o -name 'p287_*.wav' | sort -u > $tmpdir/cv.flist
+find $test_dir -name '*.wav' | sort -u > $tmpdir/tt.flist
```
Comment (Emrys365) [inline egs2/vctk_noisy/enh1/local/vctk_data_prep.sh:44]: Could you additionally check the number of wavs in each .flist file is correct?

**Comment by sw005320** [comment]
LGTM!
Thanks!

**Comment by neillu23** [comment]
Thanks! I will upload results and update the scripts according to the suggestions in the next PR.

**Pull request merged**